src/main/java/uk/ac/diamond/ispyb/api/DataCollection.java

```diff
@@ -0,0 +1,593 @@
+package uk.ac.diamond.ispyb.api;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
+
+import java.sql.Timestamp;
+
+public class DataCollection {
+    private Long Id;
+    private Long GroupId;
+    private Long dcNumber;
+    private Long apertureId;
+    private Long blSubSample;
+    private Long detectorId;
+    private Double sampleId;
+    private Long polarisationId;
+    private Long positionId;
+    private String actualCenteringPosition;
+    private Double apertureSizeX;
+    private Double averageTemperature;
+    private Double axisEnd;
+    private Double axisRange;
+    private Double axisStart;
+    private String beamShape;
+    private Double beamSizeAtSampleX;
+    private Double beamSizeAtSampleY;
+    private String bestWilsonPlotPath;
+    private Double chiStart;
+    private String comments;
+    private Double detector2Theta;
+    private Double detectorDistance;
+    private Timestamp endTime;
+    private Double exposureTime;
+    private String fileTemplate;
+    private Double flux;
+    private Double fluxEnd;
+    private Double focalSpotSizeAtSampleX;
+    private Double focalSpotSizeAtSampleY;
+    private String imgContainerSubPath;
+    private String imgDir;
+    private String imgPrefix;
+    private String imgSuffix;
+    private Double kappaStart;
+    private Long noImages;
+    private Long noPasses;
+    private Double omegaStart;
+    private Double overlap;
+    private Double phiStart;
+    private Double polarisation;
+    private Boolean printableForReport;
+    private Double resolution;
+    private Double resolutionAtCorner;
+    //private RotationAxis rotationAxis;
+    private String rotationAxis;
```

Comment:
I think we probably should use the `RotationAxis` enum if we can make that work.

Reply:
For some reason this is the one enum that doesn't work. The exception I always get is `Default conversion to RotationAxis has failed: Can't convert value '' to type RotationAxis`. I don't think it's any different from any of the other enums though.

Reply:
Hmm, could it have to do with the casing of the enum values, somehow? What if you e.g. create all the rotation axis options in lowercase (i.e. `OMEGA("omega")`, ...) and in the convert method do the comparison like 
`if (e.getValue().equals(rotationAxis.toLowerCase()))`?
If you can't make it work, then let's just make a note about it, then leave it for now and fix it in the next release.

Reply:
What bothers me about that is the convert method has a default option of INVALID if it can't find a match. The exception can probably only happen if the convert method is never run. I'm guessing it's one of those method signatures that spring should find automatically?

Reply:
An enum that works (I think) is the [ContainerStatus](src/main/java/uk/ac/diamond/ispyb/api/ContainerStatus.java) one. You could try to modify RotationAxis to become more like ContainerStatus and see if that helps?

Reply:
I'm afraid they are practically identical on the Java side. The only difference is that in the database `containerStatus` is stored as a varchar in ispyb whereas `rotationAxis` is an enum on both the Java and SQL sides.

Reply:
I'm actually not sure there is any test on whether an enum can be retrieved from ispyb. I've taken a look through the tests and I don't seem to be able to find any example of this working.